src/api/server/wizard.ts

```diff
@@ -22,19 +22,19 @@ export const getIntroSectionFromAI = async (job?: string, country?: string) => {
 };
 
 export const getWorkSectionFromAI = async (
-  employer?: string,
+  companyName?: string,
   dateTime?: string[],
   position?: string,
-  remark?: string
+  description?: string
```

Comment:
It's good that we renamed this! Because we need consistency, otherwise it confuses us, because it could be that in one place we have one names in other place others

Reply:
I've just reviewed all the changes and I see that we still have this confusion with names, because somewhere we have remark while in other places we have description. So just for right now seems to me like having a remark name it's ok, but it would be a more obvious description to rename to an AI something

Reply:
remark and description are different variables and we need them both, you create description from remark, description comes to CvViewer remark stays in the edit form